### PR TITLE
query replaced with value

### PR DESCRIPTION
**Pull request by SumanSudhir**
Fixed Issue [#34283](https://github.com/tensorflow/tensorflow/issues/34283) and #34696

### CODE REVIEW

**Comment by SumanSudhir** [comment]
@googlebot this is my first pull what to do next?

**Comment by mihaimaruseac** [comment]
Please open against `master` branch, not `r2.0`. The release process for TensorFlow is as follows:

1. We cut the release branch for the next version. Say `r2.2` for TF2.2
2. We get cherry-picks from `master` into `r2.2` to fix issues reported via testing the RC candidates
3. We release the final version, `tensorflow==2.2.0`

That's it. Observe that we never merge back release branches into master and we only pick commits from `master` only if they fix tests for the release candidates.

After time passes we might need to do a patch release for security issues. However, even in that case we only do cherry-picks from `master` and only for the commits which are relevant to the fixed issue.

Thus, in all cases, a change that comes to a release branch will get lost and we won't be able to take it.

TL;DR: please rebase/reopen agains `master`. I'm going to switch base manually but if that doesn't work I will close the PR and ask you to reopen, rebased against `master`

**Comment by mihaimaruseac** [comment]
Manually changing base didn't work. As such, closing but please reopen against `master`
